bin/rose-make-docs

```diff
@@ -203,11 +201,12 @@ version_file () {
 
     echo '{'
     # scrape filesystem for list of rose versions which have built docs
-    VERSIONS=( $(cd "${DOCS_DIR}"; echo $DOC_VERSIONS) )
+    mapfile -t VERSIONS < <(cd "${DOCS_DIR}"; echo "${DOC_VERSIONS[@]}")
     for version in "${VERSIONS[@]}"; do
         # scrape filesystem to get list of formats this version is available in
-        formats=( $(cd "${DOCS_DIR}/${version}"; echo $DOC_FORMATS) )
-        list="    \"${version}\": $(json_list ${formats[@]})"
+        mapfile -t formats \
+            < <(cd "${DOCS_DIR}/${version}"; echo "${DOC_FORMATS[@]}")
```

Comment:
When I ran ``rose make-docs`` they built cleanly & the built docs looked fine, but something about this line caused unhappiness:
```bash
Build finished. The HTML pages are in <...>/rose.git/doc/2019.01.0/html.
make: Leaving directory `<...>/rose.git/sphinx'
<...>/rose.git/bin/rose-make-docs: line 207: cd: <...>/rose.git/doc/ACKNOWLEDGEMENT.md CONTRIBUTING.md COPYING README.md bin demo etc lib licences pytest.ini rose-version sbin sphinx t tester_temp.py tox.ini usr venv: No such file or directory
```